Description: Fix FTBFS
 glibc removed obsolete, never-implemented XSI STREAMS declarations via:
 https://sourceware.org/git/?p=glibc.git;a=commit;h=a0a0dc83173ce11ff45105fd32e5d14356cdfb9c
 And so stropts.h file is not available anymore. 
 
---

--- prayer-1.3.5-dfsg1.orig/lib/os_linux.h
+++ prayer-1.3.5-dfsg1/lib/os_linux.h
@@ -9,7 +9,10 @@
 #include <signal.h>
 #include <netdb.h>
 #include <sys/file.h>
+
+#ifdef HAVE_STROPTS_H
 #include <stropts.h>
+#endif
 
 #include <sys/prctl.h>
 
